util/types.go

```diff
@@ -1,7 +1,8 @@
 package util
 
 const (
-	// DefaultRuntime is the default command to use to run the container.
+	// DefaultRuntime should no longer be used.
+	// Default runtime should come from containers.conf
 	DefaultRuntime = "runc"
```

Comment:
```suggestion
	// Deprecated: Default runtime should come from containers.conf
	DefaultRuntime = "runc"
```
Note that this change may trigger linters to complain.